.devcontainer/devcontainer.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "nvsynth",
+    "dockerComposeFile": ["../docker-compose.yml"],
+    "service": "isaac-sim",
+    "workspaceFolder": "/nvsynth",
+    "onCreateCommand": "bash /nvsynth/scripts/docker/gen_pyrightconfig.sh",
```

The devcontainer didn't forward the ports automatically for me (as described in the Readme). But works after I added this to the `devcontainer.json` file
```suggestion
    "onCreateCommand": "bash /nvsynth/scripts/docker/gen_pyrightconfig.sh",
    "forwardPorts": [
        8211,
        49100
    ],
```